docs/ref/databases.txt

```diff
@@ -719,7 +719,7 @@ you add quotes where necessary before copying a query into an SQLite shell.
 Oracle notes
 ============
 
-Django supports `Oracle Database Server`_ versions 11.1 and higher. Version
+Django supports `Oracle Database Server`_ versions 11.2 and higher. Version
```

Comment:
Oracle 11.1 is not anymore supported by Oracle - 11.1 EOL was 3 years ago, extended support ended at the end of August 2015.

To download Oracle 11.1 server you actually have to do special request through Oracle Support and download is only available for users that do have a valid Oracle Database License.

Same rationale was used to drop support < 11.1 in 1.8:
https://code.djangoproject.com/ticket/23145